README.md

```diff
@@ -521,6 +521,35 @@ metadata:
 type: kubernetes.io/tls
 ```
 
+## Post Factory Installation Updates
+
+In cases the starlingx system was already deployed once by the Deployment
+Manager with generic configurations, additional customized configurations are
+expected to be applied once again to finalize the configuration of the system.
+The Deployment Manager can read/update a configmap and orchestrate the
+configuration process accordingly.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+	name: factory-install
+	namespace: deployment
+data:
+	factory-installed: "true"  # Initial state (change to "true" if factory-installed)
+	factory-config-finalized: "false" # Optional (change to "true" if configuration is complete)
+	system-abcd-default-updated: "false" # Optional (change to "true" if resource default updated)
```

Comment:
Does this key use following convention:
`<system-name>-default-updated`?

I think documentation related to this will be helpful.

Reply:
Done